src/medCore/database/medDatabaseController.h

```diff
@@ -73,6 +73,8 @@ class MEDCORE_EXPORT medDatabaseController: public medAbstractDbController
 
     bool execQuery(QSqlQuery& query, const char* file = NULL, int line = -1) const;
 
+    void addColumnTableIfNeeded(QSqlQuery query, QString columnName);
```

Comment:
This function takes a query that is necessarily of type ```PRAGMA table_info```, and adds the columns to the series table, but the function name is too generic and doesn't inform about these constraints. I think you can avoid taking a query argument by creating your own call to ```PRAGMA table_info``` inside the function, and also the function name could be changed to ```addColumnToSeriesTableIfNeeded```.

Reply:
(actually it should even be ```addTextColumnToSeriesTableIfNeeded``` since it only adds text type columns)